src/apps/chifra/pkg/types/types_parameter.go

```diff
@@ -4,10 +4,10 @@ import "fmt"
 
 type RawParameter interface{}
 type SimpleParameter struct {
-	ParameterType string            `json:"parameterType"`
```

Comment:
I made a change in the auto code generation that might put this back to `type`. No big deal, just FYI.